sklearn/tests/test_grid_search.py

```diff
@@ -161,6 +168,11 @@ def test_grid_search():
     grid_search.decision_function(X)
     grid_search.transform(X)
 
+    # Test the transform operations
+    transformed = grid_search.transform(X)
+    reverse_transformed = grid_search.inverse_transform(transformed)
+    assert_equal(np.array_equal(reverse_transformed, X), True)
```

Comment:
Sorry, did not know those functions existed, will look more closely into your linked url.